bungeecord/build.gradle.kts

```diff
@@ -3,6 +3,10 @@ plugins {
     packetevents.`library-conventions`
 }
 
+repositories {
+    maven("https://repo.glaremasters.me/repository/public/") // brigadier 1.2.9 for BungeeCord 1.21 API
```

Comment:
https://repo.papermc.io/service/rest/repository/browse/maven-public/com/mojang/brigadier/1.2.9/

brigadier v1.2.9 also exists in papers maven repo, why are you adding an additional repository for it?